### DISCUSSION TITLE
Bug?

### DISCUSSION THREAD

**Discussion by Stronkness** [ItJustWorksTM/EiffelVis#139] [Q&A]
Hello! 

I wonder if there exist a bug in your code or something has happened during my build of the project? There doesn't appear any arrows between the different nodes. Also at some point you can't drag horizontally and sometimes when you zoom in the screen image freezes and nodes appear in different places which where they should not be.

 I have applied SSL to the nginx configuration file if that disturbs it somehow. I have not edited the .env file in the frontend directory. I will post my nginx file here with some private variables marked as ??? 

``` 
server {
    listen 80 http2;
    listen [::]:80 http2;
    server_name ????;
    return 301 https://$server_name$request_uri;
}
server {
    listen 443 ssl http2;
    listen [::]:443 ssl http2;
    ssl_certificate             /ssl/???.pem;
    ssl_certificate_key         /ssl/???.pem;
    location /api/ {
        proxy_pass http://localhost:3001/;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "Upgrade";
        proxy_set_header Host $host;
    }
    root /var/www/html/EiffelVis;
    index index.html;
    server_name ???????;
    location / {
        try_files $uri $uri/ =404;
    }
}
```

**Comment by RuthgerD** [comment]
visual glitches are not really related to nginx or other networking snafus; did these issues pop up recently?

**Reply by Stronkness** [reply]
They didnt appeared when i tried to enable SSL
